.idea/dictionaries

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
@nieuwsma I think you accidentally committed your .idea directory, and should probably be removed.

Reply:
ugh... thanks